recipe/meta.yaml

```
@@ -1,42 +1,60 @@
 {% set name = "inequality" %}
-{% set version = "1.0.0" %}
+{% set version = "1.1.2" %}
 
 package:
   name: {{ name|lower }}
```

```suggestion
  name: {{ name }}
```
I always wonder - is _lower_ keyword necessary when _name_ is already in lower?